runner/runner/util.py

```diff
@@ -57,6 +57,18 @@ def unflatten(it: Iterable[V]) -> Iterable[Iterable[V]]:
         yield (elem,)
 
 
+def flatten_maps_list(maps: List[Mapping[str, str]], key_prefix: Optional[str] = "") -> Mapping[str, str]:
+    """
+    Take a list of maps, returning a single map with every mapping found with
+    an each key name updated with the (optionally) provided key prefix
+    """
+    flat_map = dict()
```

Comment:
```suggestion
    flat_map: Mapping[str, str] = dict()
```

Mypy will probably get mad at you because it can't deduce the type here.